route_following_plugin/src/route_following_plugin.cpp

```diff
@@ -264,7 +242,7 @@ namespace route_following_plugin
         return maneuver_msg;
```

Comment:
On line 233 above there is a literal constant 0.00001.  Its value seems arbitrary, as is that of EPSILON.  Can we use the same EPSILON in all of these places?  There is another literal a little further down also.